nextflow.config

```diff
@@ -11,28 +11,77 @@ vim: syntax=groovy
  */
 
 profiles {
-    standard {
-        includeConfig 'conf/uppmax.config'
+    cfc {
+        includeConfig 'conf/cfc.config'
+    }
+    binac {
+        includeConfig 'conf/base.config'
+        includeConfig 'conf/binac.config'
     }
     local {
+        includeConfig 'conf/base.config'
         includeConfig 'conf/local.config'
     }
+    testing {
+        includeConfig 'conf/base.config'
+        includeConfig 'conf/testing.config'
+    }
+    uppmax {
+        includeConfig 'conf/uppmax.config'
+    }
 }
 
 // Capture exit codes from upstream processes when piping
-process.shell = ['/bin/bash', '-euo', 'pipefail']
+process.shell = ['/bin/bash','-e']
 
 params.outdir = './results'
+
 timeline {
   enabled = true
-  file = "${params.outdir}/NGI-ExoSeq_timeline.html"
+  file = "${params.outdir}/QBIC-ExoSeq_timeline.html"
```

Comment:
Stick with NGI if we're keeping that name

Reply:
changed in 19087e0acac3cb64d15e9ec3dbd850114a38e4d4